<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateObjectDialog</class>
 <widget class="QDialog" name="CreateObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="objectKindLabel">
     <property name="text">
      <string>Object Kind</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="propertiesTab">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="ObjectPropertiesEditor" name="objectProperties">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>507</width>
            <height>308</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="PortEditor" name="inputs">
      <attribute name="title">
       <string>Inputs</string>
      </attribute>
     </widget>
     <widget class="PortEditor" name="outputs">
      <attribute name="title">
       <string>Outputs</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectPropertiesEditor</class>
   <extends>QWidget</extends>
   <header>gui/objectproperties.hxx</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PortEditor</class>
   <extends>QWidget</extends>
   <header>gui/createobjectdialog.hxx</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateObjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
